components/prism-glimmer.js

```diff
@@ -0,0 +1,290 @@
+/* eslint-disable no-use-before-define */
+/*tri
+* Much of this implementation is copied from highlightjs-glimmer
+*
+*/
+function setup(Prism) {
```

Comment:
This will pollute the global namespace. Please use an IIFE. [Example](https://github.com/PrismJS/prism/blob/80d475e4d8ddc8c5d744a4143b3078545d72d6aa/components/prism-css.js#L1).

Reply:
why not use modules and convert to IIFE during build for CDNs

Reply:
Because of our download page. The "bundler" implemented by our download page only concatenates source files. It doesn't parse JS or anything, it's as dumb as it gets...

The "bundler" is pretty old and hasn't seen updates for years now. The only reason we haven't replaced it with a real bundler is that it just works (there haven't been any issues with it since the day I joined the project). The only inconvenience is that we have to use IIFEs.

However, this will change in v2. We plan to use ESM in our source files, so we'll need a real bundler then.